### PR TITLE
python3Packages.voluptuous-serialize: enable tests

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Switch to GitHub as source and enable tests.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 109483` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python37Packages.voluptuous-serialize</li>
    <li>python38Packages.voluptuous-serialize</li>
    <li>python39Packages.voluptuous-serialize</li>
  </ul>
</details>

File: pkgs/development/python-modules/voluptuous-serialize/default.nix [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
+, fetchFromGitHub
+, voluptuous
+, pytestCheckHook
+}:
 
 buildPythonPackage rec  {
   pname = "voluptuous-serialize";
   version = "2.4.0";
 
-  disabled = !isPy3k;
```
Comment (etu) [inline pkgs/development/python-modules/voluptuous-serialize/default.nix:7]: It doesn't build on Python2 with the following error:
```
Check whether the following modules can be imported: voluptuous_serialize
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 1, in <lambda>
  File "/nix/store/9j34pp0ajq2hzb3hi7zv0l3bpszm4n43-python-2.7.18/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/nix/store/5jv5hiimrkwld75prpxh6a4hf1zps0jy-python2.7-voluptuous-serialize-2.4.0/lib/python2.7/site-packages/voluptuous_serialize/__init__.py", line 17
    def convert(schema, *, custom_serializer=None):
                         ^
SyntaxError: invalid syntax
```

So we should keep this line.

**Review by etu — APPROVED** [review]
Other than the removal of `disabled` for python2 versions, I think this is fine.

File: pkgs/development/python-modules/voluptuous-serialize/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/voluptuous-serialize/default.nix:7]: Reverted, thanks.

**Review by mweinelt — APPROVED** [review]

**Pull request merged**